.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,41 @@
+name: Bug Report
+description: Create a report to help us improve
+title: "[BUG] - Issue Title"
+labels: bug
+
+body:
+  - type: markdown
+    attributes:
+      value: "## Describe the bug\nA clear and concise description of what the bug is."
+
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: "List the steps to reproduce the bug."
+      placeholder: "1. Go to '...'\n2. Click on '...'\n3. See error"
+
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: "Describe the behavior you expected to see."
+      placeholder: "A clear and concise description of what you expected to happen."
+
+  - type: input
+    attributes:
+      label: Environment (OS, Version, etc.)
+      description: "Please provide your operating system, version, and any relevant environment details."
+      placeholder: "e.g. Windows 10, Node.js v14"
```

Comment:
Provider java chassis version is the most important. Description and Example add Java Chassis 3.2.4